rudder-agent/SOURCES/patches/cfengine/13310-disable-openssl-engine.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/libutils/libcrypto-compat.c b/libutils/libcrypto-compat.c
-index bf39f574f..5d29b7cfb 100644
---- a/libutils/libcrypto-compat.c
-+++ b/libutils/libcrypto-compat.c
-@@ -14,7 +14,7 @@
- #if OPENSSL_VERSION_NUMBER < 0x10100000L
- 
- #include <string.h>
--#include <openssl/engine.h>
-+//#include <openssl/engine.h>
```

Comment:
Has this been removed by cfengine ?

Reply:
In fact thfile still exists, it has just changed its location into libntech/libutils/libcrypto-compat.c

Reply:
engine.h is not included anymore

Reply:
still there indeed.